components/helptext/index.css

```diff
@@ -11,95 +11,75 @@
 .spectrum-HelpText {
 	--spectrum-helptext-content-color-default: var(--spectrum-neutral-subdued-content-color-default);
 	--spectrum-helptext-icon-color-default: var(--spectrum-neutral-subdued-content-color-default);
 
-	&.spectrum-HelpText--sizeS {
-		--spectrum-helptext-min-height: var(--spectrum-component-height-75);
-		--spectrum-helptext-icon-size: var(--spectrum-workflow-icon-size-75);
-		--spectrum-helptext-font-size: var(--spectrum-font-size-75);
-		--spectrum-helptext-text-to-visual: var(--spectrum-text-to-visual-75);
-		--spectrum-helptext-top-to-workflow-icon: var(--spectrum-help-text-top-to-workflow-icon-small);
-		--spectrum-helptext-bottom-to-workflow-icon: var(--spectrum-helptext-top-to-workflow-icon);
-		--spectrum-helptext-top-to-text: var(--spectrum-component-top-to-text-75);
-		--spectrum-helptext-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
-	}
-
-	&,
-	&.spectrum-HelpText--sizeM {
-		--spectrum-helptext-min-height: var(--spectrum-component-height-75);
-		--spectrum-helptext-icon-size: var(--spectrum-workflow-icon-size-100);
-		--spectrum-helptext-font-size: var(--spectrum-font-size-75);
-		--spectrum-helptext-text-to-visual: var(--spectrum-text-to-visual-75);
-		--spectrum-helptext-top-to-workflow-icon: var(--spectrum-help-text-top-to-workflow-icon-medium);
-		--spectrum-helptext-bottom-to-workflow-icon: var(--spectrum-helptext-top-to-workflow-icon);
-		--spectrum-helptext-top-to-text: var(--spectrum-component-top-to-text-75);
-		--spectrum-helptext-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
-	}
-
-	&.spectrum-HelpText--sizeL {
-		--spectrum-helptext-min-height: var(--spectrum-component-height-100);
-		--spectrum-helptext-icon-size: var(--spectrum-workflow-icon-size-200);
-		--spectrum-helptext-font-size: var(--spectrum-font-size-100);
-		--spectrum-helptext-text-to-visual: var(--spectrum-text-to-visual-100);
-		--spectrum-helptext-top-to-workflow-icon: var(--spectrum-help-text-top-to-workflow-icon-large);
-		--spectrum-helptext-bottom-to-workflow-icon: var(--spectrum-helptext-top-to-workflow-icon);
-		--spectrum-helptext-top-to-text: var(--spectrum-component-top-to-text-100);
-		--spectrum-helptext-bottom-to-text: var(--spectrum-component-bottom-to-text-100);
-	}
+	/* Spacing */
+	--spectrum-helptext-min-height: var(--spectrum-component-height-75);
+	--spectrum-helptext-icon-size: var(--spectrum-workflow-icon-size-100);
+	--spectrum-helptext-text-to-visual: var(--spectrum-text-to-visual-75);
+	--spectrum-helptext-top-to-workflow-icon: var(--spectrum-component-top-to-workflow-icon-100);
+	--spectrum-helptext-bottom-to-workflow-icon: var(--spectrum-helptext-top-to-workflow-icon);
+	--spectrum-helptext-top-to-text: var(--spectrum-component-top-to-text-75);
+	--spectrum-helptext-bottom-to-text: var(--spectrum-component-bottom-to-text-75);
+
+	/* Typpography */
+	--spectrum-helptext-font-size: var(--spectrum-font-size-75);
+	--spectrum-helptext-font-family: var(--spectrum-default-font-family);
```

Comment:
Would using the `--spectrum-sans-font-family-stack` work here? I noticed using Assistiv Labs the font's not rendering (for Windows) and I'm wondering if that would help.

Reply:
@aramos-adobe I agree with Rise- I think we need `--spectrum-sans-font-family-stack` instead of `default-font-family`. Even in Chrome sometimes, this font isn't rendering properly: 

<img width="271" alt="Screenshot 2025-03-26 at 7 02 32 AM" src="https://github.com/user-attachments/assets/083f43ee-feb1-4b78-99e3-da5d2baae931" />

I think it's got something to do with `adobe clean` vs `adobe-clean`...I'd have to hunt for the thread!